probe-rs/src/bin/probe-rs/cmd/cargo_flash/diagnostics.rs

```diff
@@ -101,7 +101,7 @@ pub(crate) fn render_diagnostics(error: OperationError) {
             ArtifactError::MultipleArtifacts => (
                 source.to_string(),
                 vec![
-                    "Use '--bin' to specify which binary to flash.".into(),
+                    "Use '--bin' or set a default bin to specify which binary to flash.".into(),
```

Comment:
It's not immediately obvious to me that this refers to the `Embed.toml`. However, I'm not sure how we could improve wording to clarify, while not making the error message overly verbose.